File: test-python/1_Greddy/1.py
## 1. 그리디 기본

money = int(input("금액 입력: "))
coin = [500, 100, 50, 10]
count = 0
for i in coin:
    count += money // i  # //는 정수만 나오는 나눗셈 연산
    money %= i

print(count)

File: test-python/1_Greddy/1.c
#include <stdio.h>
int main()
{
	int money;
	int coin[4] = { 500, 100, 50, 10 };
	int count = 0;

	printf("금액 입력: ");
	scanf("%d", &money);
	for (int i = 0; i < 4; i++) {
		count += money / coin[i];
		money %= coin[i];
	}
	printf("%d개\n", count);
	return 0;
}